.github/workflows/tests.yml

```diff
@@ -17,7 +17,9 @@ jobs:
     - uses: actions/checkout@v2
 
     - name: update apt
-      run: sudo apt update
+      run: |
+        sudo apt update
+        sudo apt-get update
```

Comment:
Why do we run both `apt` and `apt-get`?

Reply:
I'd probably update this commit to modify all our workflows to use `apt-get`. Reading through the build fail I noticed `WARNING: apt does not have a stable CLI interface. Use with caution in scripts.`.

Reply:
I've changed it

Reply:
Cool. Fire when ready.